### PR TITLE
Add more documentation and notebooks

### PR DESCRIPTION
**Pull request by sinnec**
Edit: This PR brings the following changes:
- Chnages color to whitebox turquoise.
- (Probably) Fixes link references.
- Adds example notebooks
- Additions to docs and README
- Refactoring

Screenshot 2023-01-17 at 17 17 15

I'll add more markdown text, but how does it seem as a first iteration?

### CODE REVIEW

**Comment by sinnec** [comment]
@NickNtamp Weird because I used pip to install it. Do you believe we could combine the SDK documentation with the notebook, because their content is almost identical?

**Comment by NickNtamp** [comment]
> @NickNtamp Weird because I used pip to install it. Do you believe we could combine the SDK documentation with the notebook, because their content is almost identical?

As it is the 1st iteration, seems fine for me.

**Comment by stavrostheocharis** [comment]
I also validate that had to install the "mkdocs-jupyter" in order to run it.

To tell you the truth I don not like very much as it is depicted as notebook inside the docs (just personal view). 
Maybe if we could include the notebooks inside the repo and reference them inside the docs?
They do something similar in mlflow docs:
image

And maybe just have the code in a table as depicted below? (also to mention somewhere "Python")
image

**Comment by sinnec** [comment]
@stavrostheocharis Well, the code exists in the "Using Whitebox" section. It's almost identical (at least the one I made). I don't reject that thought of just referencing the notebooks, also given the fact that they look kinda ugly in the docs. Not all markdown can be applied on them. I made a sample one if you ewant to check out and we can decide on what we want to do and continue with that.

I personally prefer a folder with notebooks that's dettached from the docs. Your thoughts @NickNtamp @momegas ?

**Comment by sinnec** [comment]
See the sections "Using Whitebox" and the mispelled "Usaging Whitebox" under the tutorial section. They tell the same story but one looks uglier!

**Comment by NickNtamp** [comment]
> @stavrostheocharis Well, the code exists in the "Using Whitebox" section and the descriptions provided are more detailed than some comments. It's almost identical (at least the one I made). I don't reject that thought of just referencing the notebooks, also given the fact that they look kinda ugly in the docs. Not all markdown can be applied on them. I made a sample one if you ewant to check out and we can decide on what we want to do and continue with that.
> 
> I personally prefer a folder with notebooks that's dettached from the docs. Your thoughts @NickNtamp @momegas ?

As it is a first iteration i like it @sinnec. The only bizarre thing is that, ok...we do this for what reason? I mean we create a model, load a training and inference dataset and then what? I think that we have to give something at the end (e.g. use the UI, go to page http... etc.)

File: docs/mkdocs/docs/tutorial/monitors_alerts.md [hunk 1]
```diff
@@ -14,10 +14,9 @@ model_monitor = wb.create_model_monitor(
     name="test",
     status=MonitorStatus.active,
     metric=MonitorMetrics.accuracy,
-    feature="feature1",
-    lower_threshold=0.7,
     severity=AlertSeverity.high,
-    email="jaclie.chan@somemail.io",
+    email="jackie.chan@somemail.io",
+    lower_threshold=0.7
 )
```
Comment (sinnec) [inline docs/mkdocs/docs/tutorial/monitors_alerts.md:20]: Accuracy shouldn't have `feature` set!

File: whitebox/schemas/modelMonitor.py [hunk 2]
```diff
@@ -30,10 +30,10 @@ class ModelMonitorBase(BaseModel):
     name: str
     status: MonitorStatus
     metric: MonitorMetrics
-    feature: Optional[str]
-    lower_threshold: Optional[float]
     severity: AlertSeverity
     email: str
+    feature: Optional[str]
+    lower_threshold: Optional[float]
```
Comment (sinnec) [inline whitebox/schemas/modelMonitor.py:37]: I reordered them so the optional params are at the end.

**Review by NickNtamp — APPROVED** [review]
Very good work @sinnec . Some links are missing, especially in metrics etc but I can fill them, working in the other PR for the regression models. Ok from my side.

**Comment by sinnec** [comment]
I know that there are missing links. After the doc's restructure I didn't know where they should point and since I don't know if they will work, I omitted them. 😊

**Pull request merged**